src/ray/raylet/local_object_manager.cc

```diff
@@ -504,6 +505,13 @@ void LocalObjectManager::FillObjectSpillingStats(rpc::GetNodeStatsReply *reply)
   stats->set_restored_objects_total(restored_objects_total_);
 }
 
+void LocalObjectManager::RecordObjectSpillingStats() const {
+  stats::SpillingBandwidthMB.Record(spilled_bytes_total_ / 1024 / 1024 /
+                                    spill_time_total_s_);
+  stats::RestoringBandwidthMB.Record(restored_bytes_total_ / 1024 / 1024 /
+                                     restored_bytes_total_);
```

Comment:
Will fix

Reply:
Is this supposed to be bandwidth or total bytes restored?

Reply:
oh it will be spill_time_total_s

Reply:
sorry restore_time_total_s

Reply:
i see... `spill_time_total_s_` will give us the average bandwidth over the lifetime of the cluster (instead of the bandwidth at the data point) right? 

This sounds like a reasonable place to start, just want to confirm it's the goal.

Reply:
Yep that's correct. We can add more finer grained stats in the future, but it is just matching the output of ray memory --stats-only now.

Reply:
https://github.com/ray-project/ray/blob/4ad79ca963015c14ef202100c7db6ed5bd29d884/python/ray/internal/internal_api.py#L52

Reply:
Yeah, we'll probably eventually want the instantaneous ones, but for now it makes sense to export the metrics we already have.